Automattic/wp-calypso
# Plugins update manager: Schedule updates UI

Related to https://github.com/Automattic/dotcom-forge/issues/5544
Closes https://github.com/Automattic/dotcom-forge/issues/5648
Closes https://github.com/Automattic/dotcom-forge/issues/5644

## Proposed Changes

* Created the `PluginsUpdateManager` module with the supporting Schedule components:
  * `ScheduleCreate`, `ScheduleForm`, `ScheduleList`
  * Icons
  * Badge
* Integrated `PluginsUpdateManager` under the following routes:
  * `/plugins/update-manager/{SITE_SLUG}`
  * `/plugins/update-manager/create/{SITE_SLUG}`
* Added `useSitePlugins` query for listing plugin list inside the ScheduleForm

**NOTE:**
* Error validation messages and empty states are rendered by default for the review purposes
* In this iteration, we skipped integrating translate and eventRecord; it will be done in the next PRs
* Business logic and the BE integration will be covered in the next PRs

The Figma design: kjZaFpCQCfJnwpDdyeojNK-fi-4210_28659

## Testing Instructions

* Use GH direct link
* Go to `/plugins/update-manager/{SITE_SLUG}` or `/plugins/update-manager/create/{SITE_SLUG}`
* Check desktop and mobile screens and compare them with figma

<img width="1728" alt="Screenshot 2024-02-25 at 21 41 44" src="https://github.com/Automattic/wp-calypso/assets/1241413/a14d74a9-36e5-4052-8b39-c80850d51822">

<img width="1728" alt="Screenshot 2024-02-25 at 21 42 16" src="https://github.com/Automattic/wp-calypso/assets/1241413/aff54a0b-032c-43ae-9aae-7f95284c63b1">

<img width="1728" alt="Screenshot 2024-02-25 at 21 42 31" src="https://github.com/Automattic/wp-calypso/assets/1241413/b82830d9-1179-4eb8-ab7c-87ff3cbba11b">

## Pre-merge Checklist
